ProjectLighthouse/Types/Levels/Slot.cs

```diff
@@ -52,6 +52,12 @@ public class Slot
         [ForeignKey(nameof(CreatorId))]
         public User Creator { get; set; }
 
+        [NotMapped]
+        public string CreatorName
+        {
+            get => this.Creator?.Username;
+        }
+
```

Comment:
I don't really see the point of this...

Reply:
You need to include the creator in the query like so:

```csharp
this.database.Slots.Include(s => s.Creator)
```

Reply:
It doesn't include it automatically because it (rightfully) assumes you don't need it by default.

Reply:
Made things look more like the original code but with the nullable on Creator, see https://github.com/LBPUnion/project-lighthouse/pull/38/commits/ac8e1dc303f2b2d408969b8eceafe838a013b020

Reply:
Found the culprit. See https://github.com/LBPUnion/project-lighthouse/pull/38/commits/7e294b8c2110dca1fb88dd4f37a17bb33685fb50 for the missing include for queued levels.
Fixed!